mqtt.md

```diff
@@ -135,50 +130,60 @@ echo API_KEY=...
 
 <!--s-->
 ## REST tokens
+
 - can be requested over the _REST API_ of DSH on the `/auth/v0/token` endpoint
 - this endpoint requires an _API-key_ and the name of your _tenant_
 
 <!--v-->
-## Command
+### Getting a REST token
+
 ```bash
-curl -X POST \
+  curl -X POST \
```

Comment:
This shouldn't be indented as it is the first line of this multi-line command.

Reply:
Fixed.